manim/tex_font_template.tex

```diff
@@ -0,0 +1,13 @@
+\documentclass[preview]{standalone}
+
+\usepackage[english]{babel}
+\usepackage{amsmath}
+\usepackage{amssymb}
```

Comment:
Is this supposed to not have any of the other packages??

Reply:
I didn't need them here.